bluemira/equilibria/diagnostics.py

```diff
@@ -0,0 +1,25 @@
+# SPDX-FileCopyrightText: 2021-present M. Coleman, J. Cook, F. Franza
+# SPDX-FileCopyrightText: 2021-present I.A. Maione, S. McIntosh
+# SPDX-FileCopyrightText: 2021-present J. Morris, D. Short
+#
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+"""
+Diagnostic Options for use in the equilibria module.
+"""
+
+from dataclasses import dataclass
+from enum import Enum, auto
+
+
+class PicardDiagnosticType(Enum):
```

Comment:
Probably dont need the world Type?
```suggestion
class PicardDiagnostic(Enum):
```